<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>983</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Имитатор МУС200</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>965</width>
        <height>0</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>500</width>
            <height>141</height>
           </size>
          </property>
          <property name="title">
           <string>Рабочий АК</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_15">
           <item>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <layout class="QHBoxLayout" name="horizontalLayout_14" stretch="0,0,0">
               <item>
                <widget class="QLabel" name="label">
                 <property name="styleSheet">
                  <string notr="true">background-color: rgb(0, 255, 0);</string>
                 </property>
                 <property name="text">
                  <string>β тек =</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="doubleSpinBox_Betta"/>
               </item>
               <item>
                <widget class="QCheckBox" name="checkBox_Replace_Betta">
                 <property name="text">
                  <string>зам.</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="1" column="2">
              <layout class="QHBoxLayout" name="horizontalLayout_29" stretch="0,0">
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>ЗП Ст =</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="spinBox_ZPst">
                 <property name="maximum">
                  <number>1</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="1" column="1">
              <layout class="QHBoxLayout" name="horizontalLayout_17" stretch="0,0">
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>РП вкл =</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="spinBox_RPon">
                 <property name="maximum">
                  <number>1</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="1" column="0">
              <layout class="QHBoxLayout" name="horizontalLayout_18" stretch="0,0">
               <item>
                <widget class="QLabel" name="label_10">
                 <property name="text">
                  <string>Готовность =</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="spinBox_Readhiness">
                 <property name="maximum">
                  <number>1</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="2" column="3">
              <layout class="QHBoxLayout" name="horizontalLayout_32" stretch="0,0">
               <item>
                <widget class="QLabel" name="label_23">
                 <property name="text">
                  <string>Режим =</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="spinBox_Mode">
                 <property name="maximum">
                  <number>1</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="1" column="3">
              <layout class="QHBoxLayout" name="horizontalLayout_24" stretch="0,0">
               <item>
                <widget class="QLabel" name="label_12">
                 <property name="text">
                  <string>ОП Ст =</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="spinBox_OPst">
                 <property name="maximum">
                  <number>1</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="2" column="0">
              <layout class="QHBoxLayout" name="horizontalLayout_23" stretch="0,0">
               <item>
                <widget class="QLabel" name="label_11">
                 <property name="text">
                  <string>Блокир. Дв. =</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="spinBox_LockDv">
                 <property name="maximum">
                  <number>1</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="2" column="1">
              <layout class="QHBoxLayout" name="horizontalLayout_30" stretch="0,0">
               <item>
                <widget class="QLabel" name="label_21">
                 <property name="text">
                  <string>РОП =</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="spinBox_ROP">
                 <property name="maximum">
                  <number>1</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="2" column="2">
              <layout class="QHBoxLayout" name="horizontalLayout_25" stretch="0,0">
               <item>
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>КЗ ОВ =</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="spinBox_KZ_OB">
                 <property name="maximum">
                  <number>1</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="3" column="0">
              <layout class="QHBoxLayout" name="horizontalLayout_31" stretch="0,0">
               <item>
                <widget class="QLabel" name="label_22">
                 <property name="text">
                  <string>АКА ИП4 =</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="spinBox_AKA_IP4">
                 <property name="maximum">
                  <number>1</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="3" column="2">
              <layout class="QHBoxLayout" name="horizontalLayout_33" stretch="0,0">
               <item>
                <widget class="QLabel" name="label_24">
                 <property name="text">
                  <string>АКА ИП2=</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="spinBox_AKA_IP2">
                 <property name="maximum">
                  <number>1</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="3" column="3">
              <layout class="QHBoxLayout" name="horizontalLayout_34" stretch="0,0">
               <item>
                <widget class="QLabel" name="label_25">
                 <property name="text">
                  <string>АКА ИП1=</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="spinBox_AKA_IP1">
                 <property name="maximum">
                  <number>1</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="3" column="1">
              <layout class="QHBoxLayout" name="horizontalLayout_35" stretch="0,0">
               <item>
                <widget class="QLabel" name="label_26">
                 <property name="text">
                  <string>АКА ИП3 =</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="spinBox_AKA_IP3">
                 <property name="maximum">
                  <number>1</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="0" column="1">
              <layout class="QHBoxLayout" name="horizontalLayout_16" stretch="0,0">
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="styleSheet">
                  <string notr="true">background-color: rgb(255, 255, 127);</string>
                 </property>
                 <property name="text">
                  <string>BSB =</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="spinBox_BSB">
                 <property name="buttonSymbols">
                  <enum>QAbstractSpinBox::UpDownArrows</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="0" column="2">
              <layout class="QHBoxLayout" name="horizontalLayout_22" stretch="0,0">
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="styleSheet">
                  <string notr="true">background-color: rgb(255, 255, 127);</string>
                 </property>
                 <property name="text">
                  <string>R/w =</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="spinBox_R_W">
                 <property name="value">
                  <number>1</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="0" column="3">
              <layout class="QHBoxLayout" name="horizontalLayout_26" stretch="0,0">
               <item>
                <widget class="QLabel" name="label_6">
                 <property name="styleSheet">
                  <string notr="true">background-color: rgb(255, 255, 127);</string>
                 </property>
                 <property name="text">
                  <string>om = </string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="spinBox_OM"/>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Flash память</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QGroupBox" name="groupBox_3">
             <property name="title">
              <string>Запись Flash</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QPushButton" name="pushButton_SendKvitokWirteFlash">
                <property name="text">
                 <string>Отправить квиток успешной записи</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox_AutoanswerWriteFlash">
                <property name="text">
                 <string>автоответ</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_6">
             <property name="title">
              <string>Чтение Flash</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QPushButton" name="pushButton_SendReadDataFlash">
                <property name="text">
                 <string>Отправить прочитанные данные</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox_AutoanswerReadFlash">
                <property name="text">
                 <string>автоответ</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>983</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="actionDebug"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QDockWidget" name="dockWidget_Net">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string>Сеть</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_10">
     <item>
      <widget class="QGroupBox" name="groupBox_Net">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>163</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string>адрес приема данных</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_IP_4">
          <property name="text">
           <string>IP = </string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditIpAddressBind">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="inputMask">
           <string>000.000.000.000</string>
          </property>
          <property name="text">
           <string>127.0.0.1</string>
          </property>
          <property name="maxLength">
           <number>15</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_Port_Bind">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="minimum">
           <number>1024</number>
          </property>
          <property name="maximum">
           <number>54000</number>
          </property>
          <property name="value">
           <number>11000</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButtonStartNet">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Старт</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonStopNet">
         <property name="text">
          <string>Стоп</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_Output">
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string>Вывод</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QPlainTextEdit" name="plainTextEdit_log"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_Serial">
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>Com порт</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QVBoxLayout" name="verticalLayout_11">
     <item>
      <widget class="QGroupBox" name="groupBox_Serial">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>163</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string>адрес приема данных</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Порт</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_ListPorts">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Скорость</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_BaudRate">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_20">
            <property name="text">
             <string>Биты данных</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_DataBits">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_18">
            <property name="text">
             <string>Четность</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_Parity">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_19">
            <property name="text">
             <string>Стопбит</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_StopBits">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="pushButtonStart_Serial">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Старт</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonStop_Serial">
         <property name="text">
          <string>Стоп</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_Common">
   <property name="windowTitle">
    <string>Общие</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QCheckBox" name="checkBox_Autoanswer">
       <property name="text">
        <string>автоответ</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_LogCmd">
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>Журнла команд</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_5">
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QTableView" name="tableView">
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionDebug">
   <property name="text">
    <string>Отладка</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
